packages/x-components/src/views/home/Home.vue

```diff
@@ -390,8 +390,9 @@
 
                           <template #related-prompts-group="{ item: { relatedPrompts } }">
                             <RelatedPrompt
-                              class="x-flex x-flex-col x-pt-24"
+                              class="x-bg-neutral-10 x-flex x-flex-col x-gap-16 x-pt-24"
```

Comment:
Hi, maybe this background is another possible customizable style for customers, maybe we could add a `containerClass` in the component to handle this attribute, which could also be used to add the gap & other spacing

Reply:
You can already achieve this by attribute inheritance. Those classes will fall to the root element inside `RelatedPrompt`, just like is done here.